components/IdeaBox.vue

```diff
@@ -11,6 +11,35 @@
         faite pour toi ! N'oublie pas de bien respecter les consignes en
         vigueur.
       </div>
+      <div class="section-filter columns is-multiline is-mobile justify-around">
+        <div class="column is-narrow">
+          <div class="mb-xs">Je suis</div>
+          <nv-select
+            v-model="tags.person"
+            :options="filterOptions.person"
+            :select-class="filterColorClass.person"
+            small
+          />
+        </div>
+        <div class="column is-narrow">
+          <div class="mb-xs">J'ai</div>
+          <nv-select
+            v-model="tags.time"
+            :options="filterOptions.time"
+            :select-class="filterColorClass.time"
+            small
+          />
+        </div>
+        <div class="column is-narrow">
+          <div class="mb-xs">J'ai besoin de</div>
+          <nv-select
+            v-model="tags.need"
+            :options="filterOptions.needs"
+            :select-class="filterColorClass.need"
+            small
+          />
+        </div>
+      </div>
```

Comment:
en fait pour ca on pourrait faire un `v-for` non ?

Reply:
Done !